right-for-life_front/src/app/happy-stories-list/happyStoriesList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { HappyStoriesListItem } from '../happy-stories-list-item';
+
+export function HappyStoriesList(props) {
+
+  const [ happyStoriesData, setHappyStoriesData ] = useState(null);
+  const [ isStoriesDisplayed, setIsStoriesDisplayed ] = useState(false);
+
+  function handleShowStoriesButtonClick() {
+    setIsStoriesDisplayed(!isStoriesDisplayed);
+  }
+
+  useEffect(() => {
+    fetch('https://raw.githubusercontent.com/AlexeyKasaev3/softServe-academy/master/demo-3-data/news.json')
```

Comment:
i think we must wait until all data-JSONs will be added from others and then use them